string filename
float32 duration
---
bool success
string message
